Producer/cfg/data.py

```diff
@@ -195,6 +200,89 @@
 
 metSequence += process.fullPatMetSequencePuppi
 
+if egFix:
```

Comment:
Why was this introduced again? egFix is no longer a thing and is always set to False.

Reply:
That is, assuming we're going to use the latest re-reco and not 03Feb2017.

Reply:
It was still in prod.py, which is all I edited directly. Loading the config file doesn't take a long time, so I wouldn't think there's a good reason to delete that if branch by hand.

Reply:
It's not so much the speed I'm worried about but the readability of the code. Why keep a bunch of ad-hoc fixes if we're not using them?

Reply:
Let's keep this for 80X-based releases.